dev.html

```diff
@@ -2150,6 +2174,44 @@ <h2>Datatypes</h2>
             </pre>
           </div>
         </div>
+
+        The datatype can be also a data reference
+
+        <div id="tabs-23a" class="nanotabs">
+          <ul>
+            <li><a href="#tab23-1a" class="selected">YARRRML</a></li>
+            <li><a href="#tab23-2a">RML</a></li>
+          </ul>
+          <div id="tab23-1a">
+             <pre id="example-lang" class="example" title="mapping with one reference datatype">
+                mappings:
+                  person:
+                    predicateobjects:
+                      - predicates: foaf:firstName
+                        objects:
+                          value: $(firstname)
+                          datatype: $(my_datatype)
+              </pre>
+          </div>
+          <div id="tab23-2a">
+            <pre id="example-lang-rml" class="example" title="mapping with one reference datatype">
+              <#TriplesMap> a rr:TriplesMap; #person
+
+                rr:predicateObjectMap [
+                  a rr:PredicateObjectMap;
```

Comment:
Move this to the same line as `rr:predicateObjectMap` to be consistent with `<#TriplesMap>`

Reply:
this is a copy-paste from a previous example (i.e., we would need to change all examples), is it necessary now?

Reply:
Ah nvm then, no need to change them all, just stay consistent with what is already there :)

Reply:
Tracked in https://github.com/kg-construct/yarrrml-spec/issues/5